src/server_manager/types/trust_certificate.d.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Represents an additional trust anchor for a single host. */
+declare interface HostAnchor {
```

Comment:
This is a declaration for things defined in the run environment, so it's a misuse of .d.ts. It should be in a .ts file instead that the other files can depend on.

Reply:
It seems there's no elegant way to do this.  However, it turns out to be unnecessary, because the IPC system has good support for multiple arguments.